docs/LUA.md

```diff
@@ -41,8 +41,10 @@ Parameters which are not available are not displayed on the screen (i.e. the lis
 
 ## BW Lua Script Notes
 
-The Lua script for radios with a black and white screen is limited in functionality and only allows for 5 parameters to be configured.  By default, these are BindPhrase, Mode, TxPwr, RxPwr, and RxOutMode.
+The Lua script for radios with a black and white screen now also supports all parameters.  But because of screen and memory limitations it looks and behaves slightly differently than the color Lua script.
 
-If one wants to be able to change a different parameter, the 'custom param list' section in the Lua script can be updated to reference a different parameter. The parameter numbers are zero-based and can be determined from the 'Setup parameter list' section located in [setup_list.h](https://github.com/olliw42/mLRS/blob/main/mLRS/Common/setup_list.h).
+The starting screen page contains [common parameters](PARAMETERS.md#mlrs-documentation-parameters-v1300), status information, and commands such as bind and save.  Seven additional parameters appear on each subsequent page and page navigation is linear via the "prev" and "next" commands.
```

Comment:
isn't "screen page" kind of tautologous duplication ?

is there also a "non-linear2 page navigation? consider if "linear" could go

Reply:
ah,ok ... just never heard that word in that context :)